components/suggest/src/store.rs

```diff
@@ -409,7 +409,7 @@ where
                     };
                     let data = self.settings_client.get_attachment(&attachment.location)?;
                     writer.write(|dao| {
-                        dao.put_icon(icon_id, &data)?;
+                        dao.put_icon(icon_id, &data, &attachment.mimetype)?;
```

Comment:
`attachment.mimetype` looks like it's a `String`, not an `Option<String>`, so I wonder if we should make the column `TEXT NOT NULL`—WDYT?

Reply:
Yep, indeed! Thanks!